src/field_theory/ratfunc.lean

```diff
@@ -42,9 +42,13 @@ To provide good API encapsulation and speed up unification problems,
 
 We need a couple of maps to set up the `field` and `is_fraction_ring` structure,
 namely `ratfunc.of_fraction_ring`, `ratfunc.to_fraction_ring`, `ratfunc.mk` and
-`ratfunc.aux_equiv`.
+`ratfunc.to_fraction_ring_ring_equiv`.
 All these maps get `simp`ed to bundled morphisms like `algebra_map (polynomial K) (ratfunc K)`
 and `is_localization.alg_equiv`.
+
+To convert back to `fraction_ring`, use `(fraction_ring.alg_equiv (polynomial K) (ratfunc K)).symm`,
+possibly with `alg_equiv.restrict_scalars` to change the `ratfunc K ≃ₐ[polynomial K] polynomial K`
+to `ratfunc K ≃ₐ[K] polynomial K`.
```

Comment:
I don't really like the phrasing of this comment, since it is mixing implementation details (`ratfunc K` is implemented as `fraction_ring (polynomial K)` with usage hints (`ratfunc K` is `polynomial K`-equivalent to `fraction_ring (polynomial K)`). I was careful to design the API so we could switch out the quotient type `fraction_ring` with e.g. a structure type of a monic numerator + coprime denominator, and only definitions inside this file have to change.

How about expanding the comment on lines 23-24 to something like:
```markdown

You can use `is_fraction_ring` API to treat `ratfunc` as the field of fractions of polynomials:
 * `algebra_map (polynomial K) (ratfunc K)` maps polynomials to rational functions
 * `is_fraction_ring.alg_equiv` maps other fields of fractions of `polynomial K` to `ratfunc K`, in particular:
 * `fraction_ring.alg_equiv (polynomial K) (ratfunc K)` maps the generic field of fraction construction to `ratfunc K`.
   Combine this with `alg_equiv.restrict_scalars` to change the `ratfunc K ≃ₐ[polynomial K] polynomial K`
   to `ratfunc K ≃ₐ[K] polynomial K`.
```